packages/web/src/components/nav/desktop/NowPlayingArtworkTile.tsx

```diff
@@ -69,10 +69,11 @@ export const NowPlayingArtworkTile = () => {
     getTrack(state, { id: trackId })
   )
   const { doesUserHaveAccess } = usePremiumContentAccess(track)
+  const isPreviewing = useSelector(getPreviewing)
   const shouldShowPurchaseDogEar =
     track?.premium_conditions &&
     'usdc_purchase' in track.premium_conditions &&
-    !doesUserHaveAccess
+    (!doesUserHaveAccess || isPreviewing)
```

Comment:
Almost feels like maybe this should be a shared function or hook 😁

Reply:
I'm going to not do this here just because there is another spot where the logic is slightly different and I don't think it's all the bad to have it "wet"

However, I recognize that you may disagree so hit me and tell me to do it and I will :)